polish-adblock-filters/adblock.txt

```diff
@@ -3602,8 +3602,8 @@ zywiecinfo.pl##.promoted
 ||www.chwaszczyno.pl/templates/chwaszczyno*/images/designer/*_baner*.jpg$image
 ||www.comperialead.pl/upload/$image
 ||www.czarnkow.info/kir/ogl/$image
-||www.dobreprogramy.pl/cdn/*/huawei/huawei-$image 
-||www.dobreprogramy.pl/cdn/*/huawei/bg-mobile.jpg$image 
```

Comment:
@MajkiIT Co to za białe znaki na końcu linii? Nieładnie, @wmyrda się pogniewa :stuck_out_tongue:  Polecam VS Code z rozszerzeniem podświetlającym białe znaki na końcu i początku linii => `https://marketplace.visualstudio.com/items?itemName=ybaumes.highlight-trailing-white-spaces, a nawet można ustawić, by były automatycznie usuwane po zapisie :smile:.

Reply:
Wymyrdy nie ma na github od października 2018?

Reply:
Nic nie wiadomo, może wchodzi, tylko nic nie mówi.